content/en/monitoring/notification-manager.md

```diff
@@ -20,8 +20,8 @@ Supported receivers includes:
 
 ### Config Prometheus Alertmanager to send alerts to Notification Manager
 
-Notification Manager uses port `19093` and API path `/api/v2/alerts` to receive alerts sending from Prometheus Alertmanager of Kubsphere.
-To receive Alertmanager alerts, eidt the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system`, add webhook config like below to the `receivers` section of alertmanager.yaml:
+Notification Manager uses port `19093` and API path `/api/v2/alerts` to receive alerts sending from Prometheus Alertmanager of Kubesphere.
+To receive Alertmanager alerts, edit the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system`, add webhook config and route like below to the alertmanager.yaml:
```

Comment:
To receive Alertmanager alerts, edit the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system`, add webhook config and route like below to the alertmanager.yaml:
=>
To receive Alertmanager alerts, KubeSphere already added Alertmanager webhook and route configurations like below ( by editing the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system ):